### PR TITLE
Refactor scim api docs and fix response codes.

### PR DESCRIPTION
**Pull request by robfusion**

### CODE REVIEW

File: site/docs/v1/tech/lambdas/scim-group-request-converter.adoc [hunk 1]
```diff
@@ -1,7 +1,7 @@
 ---
 layout: doc
-title: SCIM Group Request Converter lambda
-description: An overview of the SCIM Group Request Converter lambda
+title: SCIM Group Request Converter Lambda
+description: An overview of the SCIM Group Request Converter Lambda
 ---
 
 :sectnumlevels: 0
```
Comment (mooreds) [inline site/docs/v1/tech/lambdas/scim-group-request-converter.adoc:7]: Example `https://login.` on line 29 below renders weird.

**Review by mooreds — CHANGES_REQUESTED** [review]
A few changes requested.

File: site/docs/v1/tech/apis/scim/_scim-response-codes.adoc [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+|Code |Description
+
+ifndef::delete[]
+|200
+|The request was successful. The response will contain a JSON body.
+endif::[]
+
+ifdef::delete[]
+|204
+|The request was successful. The response will contain a JSON body.
```
[the remark this answers is not in the document]
Comment (robfusion) [reply site/docs/v1/tech/apis/scim/_scim-response-codes.adoc]: That's from the SCIM rfc.

File: site/docs/v1/tech/lambdas/scim-group-request-converter.adoc (hunk 1 above)
Comment (robfusion) [reply site/docs/v1/tech/lambdas/scim-group-request-converter.adoc:7]: Tried something different.

File: site/docs/v1/tech/apis/scim/SCIMEnterpriseUser.adoc [hunk 3]
```diff
@@ -8,19 +8,19 @@ description: APIs for provisioning SCIM EnterpriseUsers in FusionAuth using a SC
 
 This page contains all of the APIs for managing Users through link:https://datatracker.ietf.org/doc/html/rfc7643#section-4.3[SCIM EnterpriseUser] requests.
 
-* <<Create a EnterpriseUser>>
-* <<Delete a EnterpriseUser>>
-* <<Retrieve a EnterpriseUser>>
-* <<Update a EnterpriseUser>>
+* <<Create an EnterpriseUser>>
```
Comment (mooreds) [inline site/docs/v1/tech/apis/scim/SCIMEnterpriseUser.adoc:11]: Nice catch

File: site/docs/v1/tech/lambdas/scim-group-request-converter.adoc (hunk 1 above)
Comment (mooreds) [reply site/docs/v1/tech/lambdas/scim-group-request-converter.adoc:7]: Looks good.

**Review by mooreds — APPROVED** [review]

**Pull request merged**